### DISCUSSION TITLE
how to add environment variables privately?

### DISCUSSION THREAD

**Discussion by m0x61h0x64i** [vercel/community#1033] [Help] [answered]
hi, im working on a project that is hosted in vercel and i deployed the app on a team account which is a pro account.
i wanted to know if there is a way to hide environment variables from other team members (such as admin) to not being visible in the project settings.

**Comment by amyegan** [comment, marked as answer]
Hi @m0x31h0x64i! At the moment, Pro Teams have two available roles: [Owner](https://vercel.com/docs/teams-and-accounts/team-members-and-roles#owner-role) and [Member](https://vercel.com/docs/teams-and-accounts/team-members-and-roles#member-role). Both of them have access to manage environment variables. The topic of additional Pro roles was recently brought up in https://github.com/vercel/community/discussions/1011 and was shared with the larger team. No ETA at the moment, but you can expect an announcement in the [changelog](https://vercel.com/changelog) whenever team roles are modified.
